packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -523,26 +529,49 @@ export class SwapQuoter {
             gasPrice = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
         }
         // get the relevant orders for the makerAsset
-        let prunedOrders = await this._getSignedOrdersAsync(makerAssetData, takerAssetData);
+        const orderPromises: Array<Promise<SignedOrder[]>> = [
+            this._getSignedOrdersAsync(makerAssetData, takerAssetData),
+        ];
+        if (
+            options.rfqt &&
```

Comment:
Nit but it might be nice to assign this big `&&` into a var, i.e:

```ts
const isRfqtEnabled = options.rfqt && options.rfqt.intentOnFilling && options.apiKey && this._rfqtTakerApiKeyWhitelist.includes(options.apiKey)
if (isRfqtEnabled) {
   ....
}
```

Reply:
IMHO I prefer it this way.  But I could be swayed.

Reply:
No biggie -- feel free to leave as-is